flatdata-generator/flatdata/generator/tree/builder.py

```diff
@@ -152,13 +152,31 @@ def _compute_structure_sizes(root):
             offset += int(field.type.width)
         struct.size_in_bits = offset
 
+def _check_ranges(root):
+    # First check that names are unique
+    for field in root.iterate(nodes.Field):
+        name = field.range_with_next
+        if not name:
+            continue
+        for sibling in field.parent.fields:
+            if sibling.name == name:
+                raise InvalidRangeName(name = name)
```

Comment:
I don't think you need `name =` here and below.

Reply:
If it is not named then it is positional, isn't it?

Reply:
Yes, but everything is positional except for `kwargs`.